elements/rocky-container-stackhpc/containerfiles/9-stackhpc

```diff
@@ -5,7 +5,7 @@ FROM docker.io/rockylinux/rockylinux:9
 RUN dnf group install -y 'Minimal Install' --allowerasing && \
     dnf install -y findutils util-linux cloud-init
 
-RUN sed -i "s/renderers:.*/renderers: ['network-manager']\n      activators: ['network-manager']/" /etc/cloud/cloud.cfg
+RUN sed -i "s/renderers:.*/renderers: ['network-manager']\n    activators: ['network-manager']/" /etc/cloud/cloud.cfg
```

Comment:
It would probably be safer to write to a file in `/etc/cloud/cloud.cfg.d/`, but I guess this works for now.

Reply:
So, I tried that, still what is in cloud.cfg seems to have precedence...

Reply:
Unless that was fixed in recent versions of course.